app/rest/pom.xml

```diff
@@ -787,6 +787,18 @@
         <scope>test</scope>
       </dependency>
 
+      <dependency>
+        <groupId>com.github.java-json-tools</groupId>
+        <artifactId>json-schema-validator</artifactId>
+        <version>2.2.8</version>
```

Comment:
I think we need:

    <exclusions>
      <exclusion>
        <groupId>javax.mail</groupId>
        <artifactId>mailapi</artifactId>
      </exclusion>
    </exclusions>